pom.xml

```diff
@@ -159,7 +159,7 @@
                     <version>${global_surefire_version}</version>
                     <configuration>
                         <!-- Skip the default running of this plug-in (or everything is run twice...see below) -->
-                        <argLine>@{argLine} -Xmx2048m</argLine>
+                        <argLine>-Xmx2048m</argLine>
```

Comment:
This was in for code coverage via JaCoCo. It basically overrides the empty argline from above that got deleted. Was it causing problems with running things with it there?

Reply:
The build was failing with this argLine. Though, I can't recall the exact error now as it's been a while. What's the impact of leaving it off? No code coverage for this module?

Reply:
It's the top level pom, so no code coverage at all, iirc. I haven't looked at it in awhile.  I'll play with it a bit and see what's going on

Reply:
I played with this a bit and I'm not convinced this works in master anymore, so if this was necessary to get this running, I'd rather push fixing coverage off to a follow on task.